Composer/packages/client/src/store/reducer/index.ts

```diff
@@ -516,9 +525,16 @@ const publishFailure: (title: string) => ReducerFunc = (title) => (state, { erro
 const getPublishStatus: ReducerFunc = (state, payload) => {
   // the action below only applies to when a bot is being started using the "start bot" button
   // a check should be added to this that ensures this ONLY applies to the "default" profile.
-  if (payload.target.name === 'default' && payload.endpointURL) {
-    state.botStatus = BotStatus.connected;
-    state.botEndpoints[state.projectId] = `${payload.endpointURL}/api/messages`;
+  if (payload.target.name === 'default') {
+    if (payload.status == 200 && payload.endpointURL) {
```

Comment:
Optional: In general I prefer having a centralized status codes enum if we are not already using some http library that has http status codes. more readability, harder to confuse, etc.

Reply:
sounds good, will do.